File: MPI/mpicheck/tests/functions/1.0/MPI_Group_translate_ranks.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2(10)
       INTEGER var_3
       INTEGER var_4(10)
       INTEGER var_5
        call mpi_group_translate_ranks(var_0, var_1, var_2, var_3, var_4, var_5)
       call pmpi_group_translate_ranks(var_0, var_1, var_2, var_3, var_4, var_5)
        end program main
    

File: MPI/mpicheck/tests/functions/1.0/MPI_Group_translate_ranks.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Group var_0;
    int var_1;
    const int var_2[2];
    MPI_Group var_3;
    int var_4[2];
    int ret;
    /* calls */
    ret = MPI_Group_translate_ranks(var_0, var_1, var_2, var_3, var_4);
    ret = PMPI_Group_translate_ranks(var_0, var_1, var_2, var_3, var_4);
    return 0;
}
